### PR TITLE
Refactored the invalidate session functionality.

### PR DESCRIPTION
**Pull request by deanhannigan**
## Description
Issue caused by merge of: https://github.com/Budibase/budibase/pull/5151

After pulling the above,  the server will no longer accepts requests to create a new session and become unresponsive.

### CODE REVIEW

**Comment by Rory-Powell** [comment]
Can you explain what has actually changed here? 😅 it looks like just a move

File: packages/backend-core/src/security/sessions.js [hunk 1]
```diff
@@ -14,25 +14,7 @@ function makeSessionID(userId, sessionId) {
...
-    createdAt: new Date().toISOString(),
-    lastAccessedAt: new Date().toISOString(),
-    ...session,
-    userId,
-  }
-  await client.store(makeSessionID(userId, sessionId), session, EXPIRY_SECONDS)
-}
-
-exports.invalidateSessions = async (userId, sessionIds = null) => {
+async function invalidateSessions(userId, sessionIds = null) {
```
Comment (Rory-Powell) [inline packages/backend-core/src/security/sessions.js:17]: Preference would be to keep the ES6 syntax as backend is all going that way, esp with TS refactor of this package pending

**Comment by aptkingston** [comment]
> Can you explain what has actually changed here? sweat_smile it looks like just a move

It's the `this.invalidateSessions` - attempting to reference the export using `this` is a reference either. The options are either pull it out as a separate function and reference it by plain function name (which is what is done here to be consistent with the other function `getSessionsForUser` in this file) or reference it using `exports.invalidateSessions`.

File: packages/backend-core/src/security/sessions.js (hunk 1 above)
Comment (aptkingston) [reply packages/backend-core/src/security/sessions.js:17]: That was my recommendation to keep it consistent with the other function declarations in this file which are just plain functions too (the others that aren't directly exported) - but glad to hear the backend is adopting ES6 syntax more, so we could just update the other functions to all be ES6 syntax too.

**Comment by deanhannigan** [comment]
> Can you explain what has actually changed here? 😅 it looks like just a move

There was a reference to `await this.invalidateSessions(userId)` on line 19 that threw an exception. The use of 'this' was the root of the issue. A minor refactor to declare the invalidateSessions function was needed to achieve the desired outcome. The syntax  was used to keep it in line with `getSessionsForUser`

**Review by Rory-Powell — APPROVED** [review]
Looks good! NAB: Could keep it as `exports.invalidateSessions` and just replace `this.` with `exports.`, as this will need refactored to TS eventually

**Review by aptkingston — APPROVED** [review]

**Pull request merged**